sharding/mock/validatorMock.go

```diff
@@ -1,41 +1,28 @@
 package mock
 
-import "sync"
-
 // ValidatorMock -
 type ValidatorMock struct {
-	pubKey     []byte
-	address    []byte
-	chances    uint32
-	mutChances sync.RWMutex
+	pubKey  []byte
+	chances uint32
+	index   uint32
 }
 
 // NewValidatorMock -
-func NewValidatorMock(pubKey []byte, address []byte, chances uint32) *ValidatorMock {
-	return &ValidatorMock{pubKey: pubKey, address: address, chances: chances}
+func NewValidatorMock(pubKey []byte, index uint32, chances uint32) *ValidatorMock {
```

Comment:
the order of index and chances is different than the normal NewValidator

Reply:
done

Reply:
ok